src/code42cli/cmds/devices.py

```diff
@@ -414,17 +415,14 @@ def bulk(state):
     pass
 
 
-@bulk.command(
-    name="deactivate",
-    help="""Deactivate all devices on the given list.
-            Takes as input a CSV with a 'guid' column.""",
-)
+@bulk.command(name="deactivate")
 @read_csv_arg(headers=["guid"])
 @change_device_name_option
 @purge_date_option
 @format_option
 @sdk_options()
 def bulk_deactivate(state, csv_rows, change_device_name, purge_date, format):
+    """Deactivate all devices from the provided CSV containing a 'guid' column."""
     sdk = state.sdk
```

Comment:
when deactivating a guid that does not exist, the error still shows up in the CSV output.
Looks kind of funny:

```
guid    deactivated                                                                                                                     
asdfd   False: Failure in HTTP call 404 Client Error: Not Found for url: https://console.us.code42.com/api/Computer/asdfd?idType=guid   
```

Reply:
I am guessing this is intentional, kind of neat, just a bit different than the other bulk commands

Reply:
Yeah, this was intentional to have the results be output, but I just added some better error handling in `_deactivate_device()` since we weren't catching the 404 correctly. Also validated the guid is an int before making the request since we know it's gonna fail anyway.